src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+    private String from;
+    private String to;
+    Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+    public String getType() {
```

Comment:
Perhaps you can add a ```@Override``` above your ```getType()``` method to specify that you wish to override this method from Task